app/app.go

```diff
@@ -353,25 +374,48 @@ func NewRegenApp(logger log.Logger, db dbm.DB, traceStore io.Writer, loadLatest
 	transferModule := transfer.NewAppModule(app.TransferKeeper)
 	transferIBCModule := transfer.NewIBCModule(app.TransferKeeper)
 
-	icaHostKeeper := icahostkeeper.NewKeeper(
+	app.IBCFeeKeeper = ibcfeekeeper.NewKeeper(
+		app.appCodec, app.keys[ibcfeetypes.StoreKey], app.GetSubspace(ibcfeetypes.ModuleName),
+		app.IBCKeeper.ChannelKeeper, // may be replaced with IBC middleware
+		app.IBCKeeper.ChannelKeeper,
+		&app.IBCKeeper.PortKeeper, app.AccountKeeper, app.BankKeeper,
+	)
```

Comment:
Is the fee middleware required for the controller? I think we want to add it anyways to support the option of tips for relayers but curious if it is necessary to implement for the controller.

Reply:
with the final release of ibc-go v5, it can be the channel keeper as well. fee keeper was only option before iirc.